### DISCUSSION TITLE
login issue when i try to login on github

### DISCUSSION THREAD

**Discussion by bencho-club** [community/community#28319] [Other Feature Feedback, Questions, & Ideas]
when I try to log in on GitHub with my other email id shiva850681@gmail.com/shiva935463@gmail.com, not log in. I try to forget the password both emails show me an error message on my email id
![git erro](https://user-images.githubusercontent.com/77884267/182833062-aaba6978-fdd7-41df-8f3c-edde07576e7f.jpg)
 
when I try to create an account show this error message
![git error 1](https://user-images.githubusercontent.com/77884267/182833217-09900ed9-185a-4e8d-9307-21f258bf484e.jpg)

please anyone solve my problem

**Comment by jfullstackdev** [comment]
try Support

https://support.github.com/contact/account

but I'm just wondering how were you able to post here if you can't log in ?
